setup-router/src/App.js

```diff
@@ -1,20 +1,27 @@
 import React, { Component, Fragment } from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
 import Navbar from './components/Navbar';
 import Home from './components/Home';
 import About from './components/About';
 import Contact from './components/Contact';
+import pageNotFound from './components/pageNotFound';
```

Comment:
In react Component names Start with Capital Letters. pageNotFound -> PageNotFound